### PR TITLE
feat: adds Reset Password Screen

### PR DESCRIPTION
**Pull request by am-casper**
Issue #264 resolved.

### CODE REVIEW

File: lib/presentation/reset_password/reset_password_view.dart [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                        children: [
+                          Text(
+                            'Enter your old password',
+                            style: GoogleFonts.notoSans(
+                              fontSize: 18,
+                              fontWeight: FontWeight.w600,
+                            ),
+                          ),
+                          20.toVerticalSizedBox,
+                          TextField(
```
Comment (aman-singh7) [inline lib/presentation/reset_password/reset_password_view.dart:63]: @am-casper TextField section seems quite repetitive. Abstract this part into a different widget like AppTextField and use it.

**Review by aman-singh7 — CHANGES_REQUESTED** [review]

File: lib/presentation/reset_password/reset_password_view.dart (hunk 1 above)
Comment (am-casper) [reply lib/presentation/reset_password/reset_password_view.dart]: Resolved.

**Comment by aman-singh7** [comment]
@am-casper Create a generic TextField specific to the app. Refer to [this](https://github.com/mdgspace/codephile-mobile/tree/main/lib/presentation/components/inputs).

**Comment by aman-singh7** [comment]
@am-casper are the changes done?

**Comment by am-casper** [comment]
> @am-casper are the changes done?

Yes.

**Review by aman-singh7 — APPROVED** [review]

**Pull request merged**